ssaHAL/ssa/core.py

```
@@ -42,6 +43,29 @@ def __fw_update_callback(_ssa: SSA, update_str: str):
     from machine import soft_reset
     soft_reset()
 
+def __property_update_action(ssa: SSA, msg: string, property: string):
```

Ruff (0.8.2)

Undefined name `SSA`

(F821)